lighter/callbacks/writer/table.py

```
@@ -60,7 +60,11 @@ def on_predict_epoch_end(self, trainer: Trainer, pl_module: LighterSystem) -> No
         # Save the records to a CSV file
         if trainer.is_global_zero:
             df = pd.DataFrame(self.csv_records)
-            df = df.sort_values("id").set_index("id")
+            try:
+                df = df.sort_values("id")
+            except TypeError:
+                pass
```

Ruff (0.8.0)

Use `contextlib.suppress(TypeError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(TypeError)`

(SIM105)